### ISSUE TITLE
SpotifyLocalAPIClass times out when running local example project

### ISSUE THREAD

**Issue by whereismyspaceship** [JohnnyCrazy/SpotifyAPI-NET#13]
DownloadString during SpotifyLocalAPIClass.Connect passing "http://127.0.0.1:4380/simplecsrf/token.json?&ref=&cors=&_=1423252676" fails with a standard timeout exception

System.Net.WebException occurred
  HResult=-2146233079
  Message=The operation has timed out
  Source=System
  StackTrace:
       at System.Net.WebClient.DownloadDataInternal(Uri address, WebRequest& request)
       at System.Net.WebClient.DownloadString(Uri address)
       at System.Net.WebClient.DownloadString(String address)
       at SpotifyAPI.SpotifyLocalAPI.RemoteHandler.query(String request, Boolean oauth, Boolean cfid, Int32 wait) in c:\Users\joshe\Dropbox\dev\SpotifyUtility\SpotifyAPI-NET\SpotifyAPI\SpoitfyLocalAPI\RemoteHandler.cs:line 143
  InnerException: 

Netstat says that port is open, and navigating to that url in my browser just gives me "not found"

![image](https://cloud.githubusercontent.com/assets/6990514/6086561/da548a4c-ae10-11e4-856e-1bc0eafebd1b.png)

Spotify web helper is def running

![image](https://cloud.githubusercontent.com/assets/6990514/6086576/ffd0e284-ae10-11e4-8ce9-25089dd1006a.png)

**Comment by JohnnyCrazy** [comment]
Hey,

Actually, I can't test it myself at the moment, since the SpotifyWebHelper is crashing everytime I want to make calls to it. (IDK why it happens yet)

The "Not found" message is expected, it pretty much says the WebHelper is running (else Chrome would display an error)

**Comment by gusper** [comment]
@JohnnyCrazy, based on your response I get the feeling you're seeing what I'm running into as well. I have a simple app using your library that primarily just waits for events whenever a track changes. However, it quickly starts to no longer get events as a result of SpotifyWebHelper no longer running (possibly crashing). I'm assuming this isn't something you've seen happen much previously?

**Comment by JohnnyCrazy** [comment]
@gusper 
I noticed that too, but I'm unable to find any reason why it would crash yet.

@whereismyspaceship 
Could you try these Steps:
* Close spotify and the API-Testproject
* Comment out the RunSpotifyWebhelper method in the API-Testproject
* Start spotify normally (SpotifyWebHelper should be running after this)
* Start the API-Testproject

This worked for me, maybe you are not allowed to start the WebHelper manually

**Comment by gusper** [comment]
Did some searching and mostly came up empty. However, I did find that there's a [beta desktop client](https://community.spotify.com/t5/Help-Desktop-Linux-Mac-and/Welcome-to-the-Desktop-Beta/m-p/932803/highlight/true#M98371) available now and it uses a SpotifyBetaWebHelper.exe instead. If I get some time, I might test adding support for it as well to see if the issue around SpotifyWebHelper.exe disappearing has been fixed.
